src/www/ui/page/AdminLicenseCandidate.php

```diff
@@ -116,6 +116,10 @@ protected function handle(Request $request)
     $ok = true;
     switch ($request->get('do')) {
       case 'verify':
+        if ($vars['suggest_text'] = $suggestLicense['rf_text']) {
```

Comment:
This method looks bit flawed to me as there is no guarantee that the suggested license will be the exact match for the new license.

What about capturing the error from DB as described in the issue?

Reply:
Ping to @ag4ums